examples/hacker_news_pwa/web/index.html

```diff
@@ -15,17 +15,25 @@
     * {
       box-sizing: border-box;
     }
-    html {
+    html, body {
       font-family: Roboto, sans-serif;
       font-size: 18px;
+      height: 100%;
+      margin: 0;
+      display: flex;
+      flex-direction: column;
     }
     body {
-      background: #f2f2f2;
+      background: #212121;
```

Comment:
What is this even styling? It doesn't seem to make a difference.

Reply:
When you overscroll on a Desktop machine, the body background is used to paint the overscroll region.

This makes the overscroll region match the header/footer

Reply:
But now the body also gets painted that color when you over scroll horizontally. 🤕 

How about this instead?
```css
body {
  overscroll-behavior: none;
}
```

Reply:
Actually that doesn't feel great. Honestly my preference is to leave this unchanged.

Reply:
Aha solution!

Keep your new styling, plus add to the `body` styles

```css
 overscroll-behavior-x: none;
```

Reply:
I just did this – saw no delta in the behavior on my laptop.

Does it do anything on mobile?

Reply:
Actually, it does. ` overscroll-behavior-x: none;` prevents me from swiping left/right to navigate!

Reply:
Ugh, it prevents over scrolling horizontally which then showed the dark body color on the sides and looked wonky. I didn't realized this prevented navigation 😭